extensions/amp-carousel/0.2/amp-carousel.js

```diff
@@ -208,6 +209,22 @@ class AmpCarousel extends AMP.BaseElement {
     return this.mutateElement(() => {});
   }
 
+  /**
+   * @param {!HTMLDivElement} button
+   * @param {*} onInteraction
+   */
+  setupButtonInteraction(button, onInteraction) {
+    button.onkeydown = (event) => {
```

Comment:
Nit: Curious why we use the direct onkeydown property instead of adding an event listener.

Reply:
Because while the tests in this PR were artisanally crafted, the implementation was copy/pasted from 0.1.
Will push up a new commit that updates them all.